spiketools/tests/spatial/test_occupancy.py

```diff
@@ -33,6 +33,9 @@ def test_compute_bin_edges():
     position = np.array([[1., 2., 3., 4., 5.], [0., 1., 2., 3., 4.]])
     area_range = [[1, 4], [1, 4]]
 
+    exp_x = np.array([1.0, 2.5, 4.0])
```

Comment:
This is very minor, but all other "expected" vars start with `expected`, should we change these ones from `exp_x` and `exp_y` to `expected_x` and  `expected_y`?

Reply:
Yes, good eye for consistency! I have updated all the examples you noted - thank you!